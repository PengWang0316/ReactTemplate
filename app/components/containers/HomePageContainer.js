import React, { Fragment } from 'react';


/* The component that shows the home page containt */
export const HomePageContainer = () => (
  <Fragment>
    Content goes here!
  </Fragment>
);
export default HomePageContainer;
