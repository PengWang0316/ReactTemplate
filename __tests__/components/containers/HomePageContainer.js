import React from 'react';
import renderer from 'react-test-renderer';
// import { shallow } from 'enzyme';

import { HomePageContainer } from '../../../app/components/containers/HomePageContainer';

describe('HomePageContainer', () => {
  test('snapshot', () => expect(renderer.create(<HomePageContainer />).toJSON()).toMatchSnapshot());
});
