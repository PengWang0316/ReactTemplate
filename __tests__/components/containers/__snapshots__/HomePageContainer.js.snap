// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePageContainer snapshot 1`] = `"Content goes here!"`;
